src/openrct2/ride/Track.cpp

```diff
@@ -345,8 +347,8 @@ const rct_trackdefinition FlatRideTrackDefinitions[256] = {
     { TRACK_FLAT,                   TRACK_SLOPE_DOWN_25,        TRACK_SLOPE_DOWN_25,        TRACK_BANK_NONE,        TRACK_BANK_NONE,        0                           },  // 37
     { TRACK_S_BEND,                 TRACK_SLOPE_NONE,           TRACK_SLOPE_NONE,           TRACK_BANK_NONE,        TRACK_BANK_NONE,        0                           },  // 38
     { TRACK_S_BEND,                 TRACK_SLOPE_NONE,           TRACK_SLOPE_NONE,           TRACK_BANK_NONE,        TRACK_BANK_NONE,        0                           },  // 39
-    { TRACK_VERTICAL_LOOP,          TRACK_SLOPE_DOWN_25,        TRACK_SLOPE_UP_25,          TRACK_BANK_NONE,        TRACK_BANK_NONE,        TRACK_UNKNOWN_VERTICAL_LOOP },  // 40
-    { TRACK_VERTICAL_LOOP,          TRACK_SLOPE_DOWN_25,        TRACK_SLOPE_UP_25,          TRACK_BANK_NONE,        TRACK_BANK_NONE,        TRACK_UNKNOWN_VERTICAL_LOOP },  // 41
+    { TRACK_VERTICAL_LOOP,          TRACK_SLOPE_DOWN_25,        TRACK_SLOPE_UP_25,          TRACK_BANK_NONE,        TRACK_BANK_NONE,        -48                         },  // 40
```

Comment:
are you really introducing unnamed numbers?

Reply:
Have you even taken the effort to look what this field is? It's preview_z_offset. It IS a number.

Reply:
Yes, I've seen, it's just that I found it a bit misleading that you of all people would introduce unnamed numbers and wanted to confirm